modules/openapi-generator/src/main/resources/python-flask/model.mustache

```diff
@@ -96,8 +98,8 @@ class {{classname}}(Model):
         :type {{name}}: {{dataType}}
         """
 {{#isEnum}}
-        allowed_values = [{{#allowableValues}}{{#values}}"{{{this}}}"{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]  # noqa: E501
 {{#isContainer}}
+        allowed_values = [{{#isNullable}}None,{{/isNullable}}{{#allowableValues}}{{#values}}{{#items.isString}}"{{/items.isString}}{{{this}}}{{#items.isString}}"{{/items.isString}}{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]  # noqa: E501
```

Comment:
We are tying to minimize the usage of flake8 E501 and other flake8 opt out lines.
Can you spread this to a multiline statement so we won't need # noqa: E501?

Reply:
Let's do this in a separate PR instead. I'll open an issue to track it to see if anyone has the time to improve it.

(this PR simply plots the fix from the python client template)

Reply:
Are you suggesting that we remove the E501s in a separate PR? For this one I prefer to omit the E501 for these lines. This update is adding them when they are not needed. Why add them when it throws a linting error which we can avoid?

Reply:
Created https://github.com/OpenAPITools/openapi-generator/issues/5030 for tracking.